_posts/2017-03-18-hypatia-meets-womens-energy-club_en.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Hypatia e.V. meets Women's Energy Club Kyiv
+description: Meeting on 15th / 16th of March
+header: Hypatia e.V. meets Women's Energy Club Kyiv
+---
+
+![PotatoeChild](/img/potatoechild.jpeg "PotatoeChild")
+**Women's Energy Club Ukraine** is a network of women working in energy related sectors. It aims to empower women and provide professional support. The club unites members of the Parliament of Ukraine (Energy Committee), experts, international organizations representatives as well as women from energy production sector. It is the first network for women in Ukraine. Currently the Club has about 30 members. The Club is supported by the European-Ukrainian Energy Agency & Heinrich-Böll-Foundation in Kiev.
+
+[**Hypatia Frauennetzwerk Erneuerbare Energien und Cleantech e.V.**](www.hypatia-network.de) is a non-profit organization for women working in the renewable energy sector. Similar to the Women's Energy Club it aims to empower women and provide professional support. Increasing the visibility of women in the sector is one of the major goals to reach for professional equity for women and men. Hypatia has about 30 members. The organization team works on voluntary basis.
```

Comment:
> Hypatia has about 30 members. 

How about adding some time information, e.g. 'currently' or 'as the time of writing'?
How about adding some time specifications.